connector_magento/__manifest__.py

```diff
@@ -7,6 +7,7 @@
  'version': '10.0.1.0.0',
  'category': 'Connector',
  'depends': ['account',
+             'base_technical_user',
```

Comment:
Do you plan to change something about https://github.com/OCA/server-tools/pull/858#discussion_r125591029?